testpilot/frontend/static-src/app/views/eol-view.js

```diff
@@ -0,0 +1,18 @@
+import BaseView from './base-view';
+
+export default BaseView.extend({
+  template: `<div class="eol-block"><div data-hook="ending-soon">
+<strong>This experiment is ending on <span data-hook="completedDate"></span></strong>.<br/><br/>
+After then you will still be able to use <span data-hook="title"></span> but we will no longer be providing updates or support.</div>
+<!--<div data-hook="experiment-completed">
+This experiment was retired on <span data-hook="completedDate"></span> and is no longer supported.</br>
+</div>--></div>`,
```

Comment:
Yeah, l10n is not sane in our current environment. L20N often comes around and plonks string replacements down over the work that Ampersand already did with hooks & bindings.

For example, consider [the place where we display the usage count](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/templates/experiment-page.js#L35) on the experiment page:

* It has an `data-l10n-id="userCountContainer"`
* [There's a localized string for that.](https://github.com/mozilla/testpilot/blob/master/locales/en-US/app.ftl#L136)
* Ampersand does it's thing with bindings and such there.
* But, then we have [some code that populates `data-l10n-args` with `model` properties on every DOM node with a `data-l10n-id` attribute whenever model data changes](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/views/base-view.js#L30). (Part of the reason for this is because L20N didn't have any concept of global variables for the whole page, just per-DOM-node data.) 
* So, whatever Ampersand did with hooks & bindings is replaced by what L20N does with localized strings and their placeholders

You can override `getL10nArgs` in the base page view to apply something other than model attributes, if necessary. I think you'd need to do that here, because this view isn't using a single `model` and instead has its own props.

Does all that make sense? I mean, insofar as it's terrible and nonsensical and shames me?